Dockerfile

```diff
@@ -19,13 +19,16 @@ RUN cp /etc/ironic/ironic.conf /etc/ironic/ironic.conf_orig && \
     crudini --set /etc/ironic/ironic.conf DEFAULT enabled_boot_interfaces pxe,ipxe && \
     crudini --set /etc/ironic/ironic.conf DEFAULT default_boot_interface ipxe && \
     crudini --set /etc/ironic/ironic.conf DEFAULT default_deploy_interface direct && \
+    crudini --set /etc/ironic/ironic.conf DEFAULT enabled_inspect_interfaces inspector && \
```

Comment:
Strictly speaking, we should keep `no-inspect` in the list. However, I don't think we have any drivers that do not support `inspector`, so it's fine for now.